analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -329,7 +329,7 @@ class SpdxDocumentFile(
     private fun SpdxPackage.toIdentifier() =
         Identifier(
             type = managerName,
-            namespace = originator?.withoutPrefix(SpdxConstants.ORGANIZATION).orEmpty(),
+            namespace = supplier?.withoutPrefix(SpdxConstants.ORGANIZATION).orEmpty(),
```

Comment:
In the commit message it says "more likely to match the supplier than the originator", this sounds like it would make sense to use the originator as a fallback.

Reply:
Indeed makes sense, done.